app/components/point.jsx

```diff
@@ -0,0 +1,181 @@
+// https://github.com/EnCiv/civil-pursuit/issues/23
+
+'use strict'
+import React, { useState } from 'react'
+import cx from 'classnames'
+import { createUseStyles } from 'react-jss'
+
+function Point(props) {
+  const { subject, description, vState, children, styles, className, ...otherProps } = props
+
+  const [isHovered, setIsHovered] = useState(false)
+
+  const classes = useStylesFromThemeFunction()
+
+  const handleMouseEnter = () => {
+    setIsHovered(true)
+  }
+
+  const handleMouseLeave = () => {
+    setIsHovered(false)
+  }
+
+  const childrenWithProps = React.Children.map(children?.props?.children ?? children, child => {
+    return React.cloneElement(child, {
+      isHovered: isHovered,
```

Comment:
I'm going to give you a challenge here.  The isHovered prop that you pass to the children is a working solution,  but I think there could be a more generic way of doing it.   Instead of passing the prop isHovered, pass classname set to 'hover'.  (and that's an absoluted name, not a local name in jss so when you are styling the button you need to use ".hover" see https://cssinjs.org/jss-plugin-nested/?v=v10.10.0.

Reply:
I said classname above, but it should be className

Reply:
This was done, but then there was a new comment and stuff added to the issue changing this.  Sorry it wasn't done as a a review comment - because I was CCing others.  Lets discuss in the call tomrrow.